reference/performance/execution-plan-bind.md

```diff
@@ -56,6 +82,20 @@ DROP [GLOBAL | SESSION] BINDING FOR SelectStmt
 
 This statement removes a specified execution plan binding at the GLOBAL or SESSION level. The default scope is SESSION.
 
+Generally, the binding in the SESSION scope is mainly used for test or in special situations. For a binding to take effect in all TiDB processes, you need to use the GLOBAL binding. The SESSION binding overrides the GLOBAL binding until the end of the SESSION.
+
+In the following example, the SESSION binding is dropped.
```

Comment:
@lawyerphx Please confirm L87 means the same thing as in the Chinese version.

Reply:
I was thinking about not to translate `承接上面关于 SESSION 绑定屏蔽 GLOBAL 绑定的例子，继续执行` literally because the two examples seem too far apart from each other and I can neither use an anchor. Now I translate the sentence literally, which might seem a second-best option. PTAL again.

Reply:
Well done!